writeup.md

```diff
@@ -0,0 +1,85 @@
+### NBA Shot Insights: Trends, Efficiency & Court Hotspots
+
+#### Design Rationale
+This visualization provides an interactive exploration of NBA players' shooting trends, efficiency under defensive pressure, and shot distribution on the court.
+
+#### Visual Encoding Choices
+Each chart was designed to highlight a different aspect of shooting performance
+1. Shooting Trends Over the Seasons
+    - Encoding
+        - X-axis: Seasons (Categorical)
+        - Y-axis: Shot % and Contested Shot % (Quantitative)
+        - Colors: Blue for Shot % and Orange for Contested Shot %
+    - Rationale
+        - A line chart effectively represents trends over time
+        - Dots on the lines allow for better granular insights at each season
+        - Dual lines allow comparison between general shot efficiency and defensive contest rate
+2. Shot Efficiency vs Defensive Pressure
+    - Encoding
+        - X-axis: Shot Efficiency (Quantitative)
+        - Y-axis: Contest Percentage (Quantitative)
+        - Circle Size: Consistent
+    - Rationale
+        - A scatter plot is ideal for analyzing correlations
+        - This chart allows users to understand if contested shots significantly impact efficiency
+3. Shot Hotspots Across the Court
+    - Encoding
+        - X-axis: Court coordinates of shots (Location)
+        - Y-axis: Court coordinates of shots (Location)
+        - Color: Red to Blue (Indicating shooting accuracy)
+    - Rationale
+        - Hexbin heatmaps efficiently summarize thousands of data points.
+        - The color gradient provides an intuitive sense of high-efficiency zones (blue) vs low-efficiency zones (red).
+        - The court outline preserves spatial awareness.
+
+#### Interaction Techniques
+1. Search & Filter by Player / Team  
+Users can quickly explore individual player tendencies or team shooting styles
+2. Smooth Transitions for Data Updates  
+This helps maintain visual continuity, preventing sudden jumps in the data
+3. Hover Interactions for Data Tooltips  
+This provides detailed information without cluttering the charts
+
+#### Alternatives Considered
+1. Bar Charts for Trends
+    - Line charts showcase trends better over time
+2. Shot Location as Scatter Dots
+    - Too much clutter with raw shot data, hexbin aggregation makes trends more interpretable
+3. Static Filters Instead of Search
+    - Searchable player/team selector improves usability over dropdowns
+
+#### References
+Data Sources
```

Comment:
are we missing any data sources?

Reply:
Do you want to mention this? https://github.com/shufinskiy/nba_data. He is scraping data from nba stats as is though.

Reply:
added